### ISSUE TITLE
Add a `change_count` field in diff output indicating how many changes there were

### ISSUE THREAD

**Issue by Mr0grog** [edgi-govdata-archiving/web-monitoring-processing#130]
To simplify things for other applications, it might be helpful to explicitly indicate whether the diff found any additions/removals with a boolean field in the diff response. This would be based on the type of diff, e.g. a change that only modified an HTML attribute on some element would have no changes for the text diff.

For example:

```js
{
    "diff": {/* content here */},
    "has_changes": true,
    "version": "0+untagged.293.gb17e8e9.dirty"
}
```

For the source and text diffs, this isn’t that big a deal (just look for changes where the type code isn’t `0`), but other diffs, like `html_diff_render`, require more work to detect whether there were changes. This is especially true for PDF diffs, which we don’t have but need to add.

See also this issue in the UI project: https://github.com/edgi-govdata-archiving/web-monitoring-ui/pull/156#issuecomment-347974856

**Comment by Mr0grog** [comment]
@danielballan any thoughts on this?

**Comment by lightandluck** [comment]
+1

**Comment by danielballan** [comment]
Would it be better to provide the _number_ of changes? Is it always possible to do so?

**Comment by Mr0grog** [comment]
I think it depends on the diff, but generally speaking, I don’t see why not.

I think the main thing here is that it just makes diff implementation a little less straightforward and “clean”—every diff type used to simply be the algorithm that highlighted changes, but now they’ll also have to format an envelope for their data and count the number of changes they found. Doesn’t seem like a particularly big deal, but worth pointing out.

**Comment by danielballan** [comment]
True.

**Comment by Mr0grog** [comment]
Ok, so is that a “yes” on doing this, but we want a count rather than a boolean?

If so, how’s `change_count` for the name of the field? So…

```js
{
    "diff": {/* content here */},
    "change_count": 5, /* whole number */
    "version": "0+untagged.293.gb17e8e9.dirty"
}
```

**Comment by danielballan** [comment]
Yep! I think an argument could be made for the Boolean field, but I personally prefer a count so if you are good with that let’s go that way.

**Comment by danielballan** [comment]
Closed by #136
